scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
+        "seed" [2]_.
+
+        Each row of `partial_match` specifies the indices of a pair of
+        corresponding nodes, that is, vertex ``partial_match[i, 0]`` is
+        matched to vertex ``partial_match[i, 1]``. Accordingly,
+        ``partial_match`` is an array of size ``(m , 2)``, where ``m`` is
+        less than the number of nodes.
 
     maximize : bool (default = False)
-        Gives users the option to solve the Graph Matching Problem (GMP)
-        rather than QAP. This is accomplished through trivial negation
+        Setting `maximize` to ``True`` solves the Graph Matching Problem (GMP)
+        rather than the QAP. This is accomplished through trivial negation
         of the objective function.
 
-
     options : dict, optional
-        A dictionary of solver options. All methods accept the following
-        options:
-
-            init : 2d-array (default = 'barycenter')
-                The algorithm may be sensitive to the initial permutation
-                matrix (or search position) chosen due to the possibility
-                of several local minima within the feasible region.
-                With only 1 initialization, a barycenter init will
-                likely return a more accurate permutation.
-
-                Choosing several random initializations (through
-                `init_weight` and `init_n`) as opposed to the non-informative
-                barycenter will likely result in a more accurate result at
-                the cost of higher runtime.
-
-                The initial position chosen:
-
-                "barycenter" : the non-informative "flat doubly stochastic
-                matrix," :math:`J=1*1^T /n` , i.e the barycenter of the
-                feasible region (where :math:`n` is the number of nodes and
-                :math:`1` is a ``(n, 1)`` array of ones).
-
-                If an ndarray is passed, it should have the same shape as
-                `cost_matrix` and `dist_matrix`, and its rows and columns
-                must sum to 1 (doubly stochastic).
-            init_weight : float, positive in range [0,1] (Default = None)
-                Allows the user to specify the amount of random perturbation
-                from the starting position of `init`
-                At each initialization, the initial permutation matrix
-                is some random point near :math:`J`, defined as
-                :math:`(\alpha J + (1- \alpha) K`, where :math:`\alpha`
-                is `init_weight` and :math:`K` is some random doubly
-                stochastic matrix.
-            init_n : int, positive (default = 1)
-                Number of random initializations of the starting
-                permutation matrix that the FAQ algorithm will undergo.
+        A dictionary of solver options.
+
+            init_J : 2d-array or "barycenter" (default = "barycenter")
+                The initial (guess) permutation matrix or search "position"
+                :math:`J`.
```

Comment:
Let's introduce the symbol _J_ and the term "search position" here. When I was first reading, I was not used to thinking about a matrix as a "position", so I suggest here that the permutation matrix can be thought of as a position.